### PR TITLE
feat: revamp Left Hand Nav Bar []

### PR DESCRIPTION
**Pull request by Astitva877**
## Description

Revamped Left Hand Nav Bar #746 

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [x] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

### CODE REVIEW

File: src/lib/components/sidebar/Sidebar.svelte [hunk 1]
```diff
@@ -1,12 +1,28 @@
...
+  import selectedEnvironment from "$lib/assets/environment-selected.svg";
+
+  import more from "$lib/assets/more.svg";
+  import hoveredMore from "$lib/assets/more-hovered.svg";
+  import selectedMore from "$lib/assets/more-selected.svg";
+
   import help from "$lib/assets/help.svg";
-  import settings from "$lib/assets/setting.svg";
+  import hoveredHelp from "$lib/assets/help-hovered.svg";
+  import selectedHelp from "$lib/assets/help-selected.svg";
```
Comment (itsmdasifraza) [inline src/lib/components/sidebar/Sidebar.svelte:17]: Move this file outside lib

Comment (Astitva877) [reply src/lib/components/sidebar/Sidebar.svelte]: fixed

File: src/pages/dashboard/Dashboard.svelte [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <script lang="ts">
-  import Sidebar from "$lib/components/sidebar/Sidebar.svelte";
+  import Sidebar from "@common/components/sidebar/Sidebar.svelte";
   import { Route } from "svelte-navigator";
```
Comment (itsmdasifraza) [inline src/pages/dashboard/Dashboard.svelte:3]: import from index

**Review by itsmdasifraza — APPROVED** [review]

**Review by gc-codes — APPROVED** [review]

**Pull request merged**